xbmc/platform/android/network/NetworkAndroid.cpp

```diff
@@ -246,6 +251,13 @@ CNetworkAndroid::~CNetworkAndroid()
     delete intf;
   for (auto intf : m_oldInterfaces)
     delete intf;
+
+  if (CJNIBase::GetSDKVersion() >= 24)
+  {
+    CJNIConnectivityManager connectivityManager(
```

Comment:
Both are valid ways to call the constructor, whatever you want

Reply:
It's more about what we have as suggestion on the codeguidelines rather than personal preference (no strong feelings really), see https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#55-initialization

Since you contribute often is probably a good idea to keep stuff like this in mind

Reply:
You're right, I'll change it to {} :+1: